cpp/CMakeLists.txt

```diff
@@ -155,6 +155,47 @@ if(BUILD_TESTS)
     endif(GTEST_FOUND)
 endif(BUILD_TESTS)
 
+
+###################################################################################################
+# - NVStrings -------------------------------------------------------------------------------------
+
+find_path(NVSTRINGS_INCLUDE "nvstrings"
+          HINTS "$ENV{NVSTRINGS_ROOT}/include"
+                "$ENV{CONDA_PREFIX}/include/nvstrings"
+                "$ENV{CONDA_PREFIX}/include")
+
+find_library(NVSTRINGS_LIBRARY "NVStrings"
+             HINTS "$ENV{NVSTRINGS_ROOT}/lib"
+                   "$ENV{CONDA_PREFIX}/lib")
+
+find_library(NVCATEGORY_LIBRARY "NVCategory"
```

Comment:
Are we using NVCATEGORY_LIBRARY AND NVTEXT_LIBRARY? I may have missed but just double check these are actually used.

Reply:
Not yet.  I copied this code from another project.  Brad and I discussed NVCATEGORY being next (although we might be able to do them together), not sure whether NVTEXT is appropriate - I'm unfamiliar with that part of cudf.